### PR TITLE
closes #11: create plate maps in Markdown format

### LINKED ISSUE #11: create plate maps in Markdown format
The method `Mix.table` creates a table that is useful for summarizing what to pipette, but it lists the plate wells as a comma-separated string, which is not easy to match visually to a 96-well or 384-well plate. 

Make a way to create plate maps in Markdown format that visually represent plate maps, e.g.,

```
## mix, 5.00 µl each
|     | 1    | 2   | 3   | 4   | 5   | 6   | 7   | 8   | 9   | 10   | 11   | 12   |
|-----|------|-----|-----|-----|-----|-----|-----|-----|-----|------|------|------|
| A   | mon0 |     |     |     |     |     |     |     |     |      |      |      |
| B   |      |     |     |     |     |     |     |     |     |      |      |      |
| C   |      |     |     |     |     |     |     |     |     |      |      |      |
| D   | mon3 |     |     |     |     |     |     |     |     |      |      |      |
| E   | mon4 |     |     |     |     |     |     |     |     |      |      |      |
| F   |      |     |     |     |     |     |     |     |     |      |      |      |
| G   |      |     |     |     |     |     |     |     |     |      |      |      |
| H   |      |     |     |     |     |     |     |     |     |      |      |      |
```
It should be customizable to put an arbitrary string in each well instead of the strand name:
```
## 5 monomer synthesis, 5.00 µl each
|     | 1   | 2   | 3   | 4   | 5   | 6   | 7   | 8   | 9   | 10   | 11   | 12   |
|-----|-----|-----|-----|-----|-----|-----|-----|-----|-----|------|------|------|
| A   | X   |     |     |     |     |     |     |     |     |      |      |      |
| B   |     |     |     |     |     |     |     |     |     |      |      |      |
| C   |     |     |     |     |     |     |     |     |     |      |      |      |
| D   | X   |     |     |     |     |     |     |     |     |      |      |      |
| E   | X   |     |     |     |     |     |     |     |     |      |      |      |
| F   |     |     |     |     |     |     |     |     |     |      |      |      |
| G   |     |     |     |     |     |     |     |     |     |      |      |      |
| H   |     |     |     |     |     |     |     |     |     |      |      |      |
```

### PR DESCRIPTION
**Pull request by dave-doty**

### CODE REVIEW

**Comment by dave-doty** [comment]
Converting this PR to draft while I work out the code formatting issues and look into the bug Damien found with `Mix.names` being the wrong type (list of lists of strings instead of list of strings).

**Comment by cgevans** [comment]
Formatting with black and rebasing on current v2 should resolve the problems.

File: src/alhambra/mixes.py [hunk 1]
```diff
@@ -43,8 +43,9 @@
 
 import warnings
 
-from pint.quantity import Quantity
+import scadnano as sc
```
Comment (cgevans) [inline src/alhambra/mixes.py]: If we need scadnano (and it looks like a non-release version?), could you add the right url/version requirement to setup.cfg?

Comment (cgevans) [reply src/alhambra/mixes.py]: Or could these use WellPos?

Comment (dave-doty) [reply src/alhambra/mixes.py]: > If we need scadnano (and it looks like a non-release version?), could you add the right url/version requirement to setup.cfg?

I was just being lazy, and I thought for some reason scadnano was already a dependency. I can change it so it doesn't depend on scadnano.

**Comment by cgevans** [comment]
It is. I don't know if the problem here is a version/branch (does it need dev?) one or something on my laptop. I'll have to check.

10 Feb 2022 14:07:27 David Doty ***@***.***>:

**Comment by cgevans** [comment]
It's working for me now; I can go ahead and merge unless you want to add something else. Do you have a preference regarding squashing/rebasing?

**Comment by dave-doty** [comment]
> It's working for me now; I can go ahead and merge unless you want to add something else. Do you have a preference regarding squashing/rebasing?

Great! No preference.

**Review by cgevans — APPROVED** [review]

**Pull request merged**